### PR TITLE
Update velocity templates

### PR DESCRIPTION
**Pull request by isuruh15**

### CODE REVIEW

File: native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm [hunk 1]
```diff
@@ -54,5 +59,5 @@ searchParameters: [#foreach($iteration in $range)#set($map_index = $iteration+1)
     ${importIdentifier}FHIRImplementationGuide baseImplementationGuide = new(baseIgRecord);
     check fhirRegistry.addImplementationGuide(baseImplementationGuide);
 
-    log:printInfo("FHIR R4 ${igCode} Module initialized.");
-}
+    log:printDebug("FHIR R4 ${igCode} Module initialized.");
```
Comment (AnjanaSenanayake) [inline native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm:62]: This was a printInfo as it shows that the package is initialized for developer. Should we change it?

File: native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm [hunk 2]
```diff
@@ -54,5 +59,5 @@ searchParameters: [#foreach($iteration in $range)#set($map_index = $iteration+1)
...
+}
```
Comment (AnjanaSenanayake) [inline native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm:63]: EoF missing

File: native/fhir-to-bal-lib/src/main/resources/templates/package.vm [hunk 3]
```diff
@@ -27,110 +30,34 @@ This package includes, FHIR ${igName} Resource types.
...
-```
 #foreach($profile in $profiles)
-[m${foreach.count}]: https://lib.ballerina.io/$org/$packageName/$packageVersion/#$profile.getResourceType()
+[m${foreach.count}]: https://lib.ballerina.io/$org/$packageName/$packageVersion#$profile.getResourceType()
 #end
 
 #foreach($profile in $profiles)
 [s$foreach.count]: $profile.getProfile()
-#end
+#end
```
Comment (AnjanaSenanayake) [inline native/fhir-to-bal-lib/src/main/resources/templates/package.vm:63]: EoF

Comment (isuruh15) [reply native/fhir-to-bal-lib/src/main/resources/templates/package.vm]: fixed in [4b041a1](https://github.com/ballerina-platform/fhir-tools/pull/7/commits/4b041a18116679fc22bc9cf8a80813bdaed0e8ee)

File: native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm (hunk 2 above)
Comment (isuruh15) [reply native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm]: fixed in [4b041a1](https://github.com/ballerina-platform/fhir-tools/pull/7/commits/4b041a18116679fc22bc9cf8a80813bdaed0e8ee)

File: native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm (hunk 1 above)
Comment (isuruh15) [reply native/fhir-to-bal-lib/src/main/resources/templates/initializer.vm]: fixed in [4b041a1](https://github.com/ballerina-platform/fhir-tools/pull/7/commits/4b041a18116679fc22bc9cf8a80813bdaed0e8ee)

**Review by AnjanaSenanayake — DISMISSED** [review]

File: ballerina/src/main/resources/health-tool-ballerina/BalTool.toml [hunk 4]
```diff
@@ -20,10 +20,10 @@ path = "resources/fhir-to-bal-lib-${version.health.cli}.jar"
 path = "resources/org.hl7.fhir.r4-4.2.0.jar"
```
Comment (sameeragunarathne) [inline ballerina/src/main/resources/health-tool-ballerina/BalTool.toml:20]: Shall we parameterize these versions as well on the other dependent jars

File: native/fhir-to-bal-lib/src/main/resources/templates/fhir_resource.vm [hunk 5]
```diff
@@ -1,16 +1,21 @@
-// Copyright (c) ${licenseYear}, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```
Comment (sameeragunarathne) [inline native/fhir-to-bal-lib/src/main/resources/templates/fhir_resource.vm:1]: Let's have the licenceyear parameterized as earlier

Comment (isuruh15) [reply native/fhir-to-bal-lib/src/main/resources/templates/fhir_resource.vm]: fixed in [882b0a4](https://github.com/ballerina-platform/fhir-tools/pull/7/commits/882b0a475c3750041e42e9d35992d5893690efde)

File: ballerina/src/main/resources/health-tool-ballerina/BalTool.toml (hunk 4 above)
Comment (isuruh15) [reply ballerina/src/main/resources/health-tool-ballerina/BalTool.toml]: fixed in [882b0a4](https://github.com/ballerina-platform/fhir-tools/pull/7/commits/882b0a475c3750041e42e9d35992d5893690efde)

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**